hooks/cache_location.py

```diff
@@ -96,22 +124,44 @@ def bundle_cache(self, project_id, pipeline_configuration_id, bundle):
         which needs to cache things per-instance can implement this using a sub
         folder inside the bundle cache location).
 
+        ..note:: This method may be slightly confusing given the use of the term
+                 "bundle_cache" throughout core which refers to the location on disk
+                 where bundles (apps, engines, frameworks) are installed. A better
+                 name for this method might have been `bundle_data_cache`. The name
+                 remains to avoid breaking client code.
```

Comment:
But the name is different now! Can we change it????  ![image](https://cloud.githubusercontent.com/assets/2604646/16160710/8f671fcc-3497-11e6-97d8-1ea530bd68a0.png)